activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -140,6 +140,7 @@ def initialize(pool_config)
         @checkout_timeout = db_config.checkout_timeout
         @idle_timeout = db_config.idle_timeout
         @size = db_config.pool
```

Comment:
We should rename `size` in `max_size`

Reply:
done. I added an alias for `:size` as well, since lots of existing code will be using that.